tests/integration/test_plugins.py

```diff
@@ -253,3 +253,39 @@ async def test_blueocean_plugin(ops_test: OpsTest, unit_web_client: UnitWebClien
     assert (
         res.status_code == 200
     ), f"Failed to access Blueocean frontend, {str(res.content, encoding='utf-8')}"
+
+
+async def test_thinbackup_plugin(ops_test: OpsTest, unit_web_client: UnitWebClient):
+    """
+    arrange: given a Jenkins charm with thinbackup plugin installed and backup configured.
+    act: when a backup action is run.
+    assert: the backup is made on a configured directory.
+    """
+    await install_plugins(ops_test, unit_web_client.unit, unit_web_client.client, ("thinBackup",))
+    backup_path = "/srv/jenkins/backup/"
+    res = unit_web_client.client.requester.post_url(
+        f"{unit_web_client.web}/manage/thinBackup/saveSettings",
+        data={
+            "backupPath": backup_path,
+            "fullBackupSchedule": "",
+            "diffBackupSchedule": "",
+            "nrMaxStoredFull": -1,
+            "excludedFilesRegex": "",
+            "forceQuietModeTimeout": 120,
+            "failFast": "on",
+            "Submit": "",
+        },
+    )
+    res.raise_for_status()
+    res = unit_web_client.client.requester.get_url(
+        f"{unit_web_client.web}/manage/thinBackup/backupManual"
+    )
+    unit_web_client.client.requester._get_crumb_data()
+    res.raise_for_status()
+
+    ret, stdout, stderr = await ops_test.juju(
+        "ssh", "--container", "jenkins", unit_web_client.unit.name, "ls", backup_path
+    )
+    assert ret == 0, f"Failed to ls backup path, {stderr}"
+    # The backup folder is of format FULL-<backup-date>
```

Comment:
Can you add this comment to the assertion instead?

Reply:
Done!